docassemble/209aPlaintiffMotionToModify/data/questions/209a_plaintiff_s_motion_to_modify.yml

```diff
@@ -202,11 +202,11 @@ fields:
 ---
 template: translate_word_modify
```

Comment:
We should remember to fix the related `if` statements. Is that an issue yet?

Reply:
Leave this alone for now, but [line 215](https://github.com/SuffolkLITLab/docassemble-209aPlaintiffMotionToModify/pull/52/files#diff-221941e601213702410f2e2f937dc7f0161f7af6f127393555a11cdc044a540aR215) still has "modify" in it. Good sign that we definitely need to change that code 👍